.travis.yml

```diff
@@ -11,6 +11,7 @@ before_install:
 install:
   - cd lambda/custom
   - yarn
+  - cd ../..
```

Comment:
I was actually going to suggest adding a `package.json` to the repo root as well and having it alias the commands in `lamdba/custom` so the `yarn` stuff can be run from one place (and the same place as `ask deploy`). But that can come in another PR (:

Reply:
Great idea!  I'll do some experimenting with it.